src/pages/Search/SentenceSearch.vue

```diff
@@ -342,28 +350,30 @@ export default {
 .ais-MenuSelect {
 	display: inline-block;
 	border-bottom: 1px dashed #118aec;
-	margin-bottom: rem-calc(10);
+	margin-bottom: rem-calc(24);
 
 	@include breakpoint(large) {
-		margin-bottom: unset;
+		margin-bottom: rem-calc(16);
 	}
 }
 
 .ais-MenuSelect select {
 	border: none;
 	background-color: transparent;
 	color: $kiva-accent-blue;
-	font-size: 1.5rem;
+	font-size: 1.375rem;
```

Comment:
I have a feeling this size will be added to stylguide. It's also used a lot in the donation lightbox.